src/System.Runtime.Extensions/src/System/Environment.cs

```diff
@@ -61,41 +61,7 @@ public static void SetEnvironmentVariable(string variable, string value, Environ
             EnvironmentAugments.SetEnvironmentVariable(variable, value, target);
         }
 
-        public static string CommandLine
```

Comment:
Environment.CommandLine on corefx is already a simulacrum. The one on the full framework returns the actual unprocessed command line (which the app can even affect after main() has started by scribbling into the buffer returned by Kernel32.GetCommandLine()). CoreFx's version is a lossy re-creation at best. I think, though, that a recreation that at least tokenizes back to the array the app got through main() is more likely to be compatible than not, no?

I cannot find any tests for CommandLine. Unfortunately, it's a bit involved to write one given our current infrastructure. We can't just use RemoteInvoker since the command line is also its channel for passing test arguments and the delegate. It probably means launching another TestApp.exe (or repurposing the one that's there.) By switching Environment.CommandLine to use Paste(), we're actually introducing some de-facto test by virtue of the fact that RemoteInvoke uses that same routine to format every argument passed to it.